modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuilder.java

```diff
@@ -100,6 +101,7 @@ class IndexBuilder implements ManuallyCloseable {
      *      to the replica.
      */
     public void scheduleBuildIndex(
+            int zoneId,
```

Comment:
Probably we should introduce a new class for these parameters, but it is not necessary to do this now.

Reply:
:ok_hand: